### ISSUE TITLE
[Feature Request] Add option to delay activation.

### ISSUE THREAD

**Issue by odysseywestra** [futo-org/voice-input#87]
Yeah, it would be nice to have a delay option for  activating the voice keyboard because when I'm listening to music or a podcast it tends to pick up the last few words before Starting with my dictation.

That I know it's not an issue with Google voice text because you have to wait for it to connect before it allows you to dictate.

Regardless, I love the app and I am really looking forward to seeing where it goes.
